src/detect-dce-stub-data.c

```diff
@@ -91,20 +91,14 @@ static InspectionBuffer *GetDCEData(DetectEngineThreadCtx *det_ctx,
     InspectionBuffer *buffer = InspectionBufferGet(det_ctx, list_id);
     if (buffer->inspect == NULL) {
         uint32_t data_len = 0;
-        uint8_t *data = NULL;
+        const uint8_t *data = NULL;
+        uint8_t endianness;
 
-        DCERPCState *dcerpc_state = txv;
-        if (dcerpc_state == NULL)
+        if (rs_dcerpc_get_stub_data(txv, &data, &data_len, &endianness, flow_flags) != 1 ||
```

Comment:
if rs_dcerpc_get_stub_data can only return 1, whats the point of this `!= 1` check?